### PR TITLE
feat: [MSSQL] expose character_set_name

### PR DESCRIPTION
**Pull request by fnaranjo-vmw**
[#185201378](https://www.pivotaltracker.com/story/show/185201378)

Related PRs:
- https://github.com/pivotal-cf/docs-csb-aws/pull/301

### Checklist:

* [x] Have you added Release Notes in the docs repositories?
* [x] Have you ran `make run-integration-tests` and `make run-terraform-tests`?
* [ ] Have you ran acceptance tests for the service under change?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: terraform-tests/mssql_test.go [hunk 1]
```diff
@@ -121,6 +122,16 @@ var _ = Describe("mssql", Label("mssql-terraform"), Ordered, func() {
 		})
 	})
 
+	Context("simple pass-through fields for aws_db_instance", func() {
```
Comment (zucchinidev) [inline terraform-tests/mssql_test.go:125]: praise: great!! thank you very much

File: aws-mssql.yml [hunk 2]
```diff
@@ -263,6 +263,12 @@ provision:
       Allow minor version upgrades automatically during the maintenance window.
       If `auto_minor_version_upgrade` is enabled, you must specify a major engine version.
     default: true
+  - <<: *nullable_string
```
Comment (zucchinidev) [inline aws-mssql.yml:266]: question: nullable means it is possible to set from string to null, but at the same time we are setting the value as `prohibit_update`, could we use string without default value?  should it have the same effect?

**Review by zucchinidev — APPROVED** [review]
I added a question Fernando. Other than that, great job!

File: aws-mssql.yml (hunk 2 above)
Comment (fnaranjo-vmw) [reply aws-mssql.yml:266]: good catch!

File: aws-mssql.yml [hunk 3]
```diff
@@ -263,6 +263,12 @@ provision:
...
+    field_name: character_set_name
```
Comment (fnaranjo-vmw) [inline aws-mssql.yml:267]:
```suggestion
  - field_name: character_set_name
    type: string
```

**Pull request merged**